spynl/main/endpoints.py

```diff
@@ -15,6 +15,7 @@
 from pyramid.security import NO_PERMISSION_REQUIRED
 
 from spynl.main.utils import get_user_info
+from spynl.main.exceptions import SpynlException
```

Comment:
good catch, it is no longer needed. I put it there while working on this but forgot to delete it. I'll update the PR.